script.js

```diff
@@ -0,0 +1,58 @@
+let books = [];
+
+const addBookButton = document.querySelector('button');
+addBookButton.onclick = function addBookButton() {
+  if (localStorage.getItem('savedArray') != null) {
+    books = JSON.parse(localStorage.getItem('savedArray'));
+  }
+  if (document.querySelector('.title').value !== '' && document.querySelector('.author').value !== '') {
+    const btitle = document.querySelector('.title').value;
+    const bauthor = document.querySelector('.author').value;
+    const book = {
+      title: btitle,
+      author: bauthor,
+    };
+    books.push(book);
```

Comment:
* This is an OPTIONAL suggestion as it is not a requirement for this project but i think in your spare time you can work on this.
* This method `push` also mutates the array of books you could use other array methods that returns a new array instead. [Read more on why it is not advisable](https://blog.sapegin.me/all/avoid-mutation/)